app/assets/stylesheets/_bootstrap-custom.scss

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Bootstrap v4.3.1 (https://getbootstrap.com/)
+ * Copyright 2011-2019 The Bootstrap Authors
+ * Copyright 2011-2019 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ */
```

Comment:
If this is upstream code shouldn't it be in `vendor` instead of `app`? I guess in truth it's a kind of halfway house in that this is a template that you're expected to edit?

Is this expected to go away once we are able to just enable the whole thing?

Reply:
Yes, it's a temporary thing. The [guidelines suggest copy+pasting](https://github.com/twbs/bootstrap-rubygem/blob/ef343b781ca1316ba3953295222578a65bd576c0/README.md#sass-individual-components) that file and working though it.